packages/scene-composer/src/utils/sceneResourceUtils.ts

```diff
@@ -55,10 +56,10 @@ export const getSceneResourceInfo = (target: string | undefined): SceneResourceI
     if (mapKey) {
       type = map[mapKey];
       const newValue = target.substring(mapKey.length);
-
+      const statuses = tagStyle ? DefaultAnchorStatusCustom : DefaultAnchorStatus;
```

Comment:
shouldn't need to check flag here. should just gate the entry point, and other part just use the value passed in to simplify